cluster-provision/images/vm-image-builder/test-containerdisk.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -ex
+
+PROJECT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE/../")"
+  pwd
+)"
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE")"
+  pwd
+)"
+
+function cleanup() {
+    echo "[test-containerdisk] cleanup"
+    if [ -z $DEBUG ]; then
```

Comment:
"DEBUG" merits an explanation in the commit or here.

Reply:
I will add it to the commit message
Basically DEBUG lets you keep the cluster created by this script, the VMI with the customized image and the customized image tar file,
So you could debug the image/VM if there was an error during the script invocation, for example:
 if the VMI that created with the new image wont be ready, you could debug what happen to the VMI and console to the VMI and check.